pkg/proxy/iptables/proxier.go

```diff
@@ -291,7 +304,7 @@ func CleanupLeftovers(ipt utiliptables.Interface) (encounteredError bool) {
 		// Hunt for service and endpoint chains.
 		for chain := range existingNATChains {
 			chainString := string(chain)
-			if strings.HasPrefix(chainString, "KUBE-SVC-") || strings.HasPrefix(chainString, "KUBE-SEP-") {
+			if strings.HasPrefix(chainString, "KUBE-SVC-") || strings.HasPrefix(chainString, "KUBE-SEP-") || strings.HasPrefix(chainString, "KUBE-XLB-") {
```

Comment:
Girish can you fix that here?

On Aug 18, 2016 6:02 PM, "Minhan Xia" <notifications@github.com> wrote:

> In pkg/proxy/iptables/proxier.go
> <https://github.com/kubernetes/kubernetes/pull/29409#discussion_r75414545>
> :
>
> > @@ -291,7 +304,7 @@ func CleanupLeftovers(ipt utiliptables.Interface) (encounteredError bool) {
> >  		// Hunt for service and endpoint chains.
> >  		for chain := range existingNATChains {
> >  			chainString := string(chain)
> > -			if strings.HasPrefix(chainString, "KUBE-SVC-") || strings.HasPrefix(chainString, "KUBE-SEP-") {
> > +			if strings.HasPrefix(chainString, "KUBE-SVC-") || strings.HasPrefix(chainString, "KUBE-SEP-") || strings.HasPrefix(chainString, "KUBE-XLB-") {
>
> ah... I missed that.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/29409/files/378b6159b1abe46cb83edc92522a884dd5b25366#r75414545>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVJdcMdUIG0qoTKDuTburxcN8Pl49ks5qhQCqgaJpZM4JSUXH>
> .
>